### DISCUSSION TITLE
`隐藏分区栏`从`简化首页`中拆分出来

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#4075] [功能建议 / Ideas]
现在新版分区首页似乎已经全量推送（开无痕还能找到个`返回旧版`按钮，但似乎点了之后毫无区别，似乎已成摆设；登录后就没看到这按钮了；除了番剧和国创）。
除了番剧国创电影电视剧综艺纪录片这几个版权内容相关的分区，各分区首页都有分区栏（额旧版好像也都有），而`简化首页`也能隐藏掉分区首页的这些分区栏，那是否应该拆分成像`隐藏顶部横幅`一样的独立功能？既以防用户以为只能隐藏首页的分区栏，又以防用户找不到该功能。

由 https://github.com/the1812/Bilibili-Evolved/discussions/1481#discussioncomment-288477 逆向思维而来。

**Comment by Geeyun-JY3** [comment]
既然现在首页的双重顶栏包含一个分区栏，本功能是否也要一并隐藏掉呢？是否有必要给个选项呢（目前`自定义顶栏`会直接隐藏掉：#3435）？
顺便提醒 #3435 可以关了。

**Reply by the1812** [reply]
这个目前还没拆出来吧, 算是跟着简化首页一起弃用了

**Reply by Geeyun-JY3** [reply]
对啊，只是看 #5038 实现了就想捞一捞这个 issue，顺便也想到了首页的双重顶栏也涉及到了这个分区栏。
